server/views/components/button-group/template.njk

```diff
@@ -0,0 +1,6 @@
+<div class="govuk-button-group button-group">
+    <button class="govuk-button action-button">{{buttonText}}</button>
```

Comment:
i don't know this service well, sorry.

Is it ok that the button is defined a little differently [here](https://github.com/ministryofjustice/prepare-a-case/blob/main/server/views/components/button/template.njk) ? or should it use button-group too?

` <button id={{id}} type={{type}} class="action-button {{classes | join(' ')}}">{{text}}</button>`

Reply:
I would definitely like an institutional opinion on that. I went with the "minimum change required" approach in the first pass. Perhaps I should look at using that button inside this template to we still have one definition of a button component. 🤔

Reply:
Although, I have just dug for other references of that macro and it is only being imported in one file where it is not referenced and if I do use it, it throws a render error so something is going on here... 🙃

Reply:
no worries, happy for you to ignore this comment 👍🏾